data/pigui/baseui.lua

```diff
@@ -0,0 +1,71 @@
+local Engine = require 'Engine'
+local pigui = Engine.pigui
+local ui = require 'pigui.libs.forwarded'
+require 'pigui.libs.wrappers'
+
+
+local defaultTheme = require 'pigui.themes.default'
+
+local pi = 3.14159264
+local pi_2 = pi / 2
+local pi_4 = pi / 4
+local two_pi = pi * 2
+local standard_gravity = 9.80665
+local one_over_sqrt_two = 1 / math.sqrt(2)
+
+ui.theme = defaultTheme
+ui.twoPi = two_pi
+ui.pi_2 = pi_2
+ui.pi_4 = pi_4
+ui.pi = pi
+
+ui.anchor = { left = 1, right = 2, center = 3, top = 4, bottom = 5, baseline = 6 }
+
+
+function ui.get_icon_tex_coords(icon)
+	assert(icon, "no icon given")
+	local count = 16.0 -- icons per row/column
+	local rem = math.floor(icon % count)
+	local quot = math.floor(icon / count)
+	return Vector2(rem / count, quot/count), Vector2((rem+1) / count, (quot+1)/count)
+end
```

Comment:
This file is using two-space indentation; if you could convert it to one-tab that'd be excellent!

Reply:
Weird, it definitely has tabs (CR, LF, TAB):
![image](https://user-images.githubusercontent.com/7342077/94460037-243aa280-0186-11eb-89c4-312d9891cc07.png)

Reply:
Alright, that's on Github then. Thank you for looking into it so closely!